client/latis.py

```diff
@@ -16,7 +40,8 @@ def data(baseUrl, latis3, dataset, returnType, operations=[]):
     else:
         return None
 
-def download(baseUrl, latis3, dataset, filename, fileFormat, operations=[]):
+def download(baseUrl, dataset, filename, fileFormat, operations=[], preferVersion2=False):
+    latis3 = __datasetCanUseVersion(preferVersion2)
```

Comment:
I like the `preferVersion2` approach that defaults to False. That way you still get a dataset even if it's from v3 instead of v2 or vice versa.

I don't think `__datasetCanUseVersion()` is quite the right name to use, however. As it's written, the boolean this function returns represents whether latis3 will be used. So something like `__datasetWillUseVersion3()` would be more correct?